server/handler/alert_services/aws_sns.go

```diff
@@ -0,0 +1,22 @@
+package alertservices
+
+import (
+	"main/server/request"
+	"main/server/response"
+
+	awssns "main/server/services/alert_service/aws_sns"
+	"main/server/utils"
+
+	"github.com/gin-gonic/gin"
+)
+
+func AwsTextMessaging(ctx *gin.Context) {
```

Comment:
More often than not, this service will be used by other services/clients, for boilerplate we can code it that way

Reply:
Sir, can you please explain a bit more?

Reply:
Right now this is being invoked by an api call right? But usually these are called via other controllers/services. It's fine you can leave it as is.

Reply:
Okay sir